src/collections/client.ts

```diff
@@ -19,18 +19,17 @@ import _ from 'lodash';
 
 interface ClientOptions {
   applicationToken: string;
-  baseApiPath: string;
+  baseApiPath?: string;
```

Comment:
Why does `baseApiPath` need to be optional?

Reply:
There is a field called `baseUrl` which is mandatory. `baseApiPath` is not used in the httpClient at the moment, so I made this optional.
I just left it as a placeholder for now, since this might come back based on the server URL parsing logic in the future.